docs(hcp): add HCP organization constraints documentation (#2535)
hashicorp/web-unified-docs: content/hcp-docs/content/docs/hcp/admin/orgs.mdx

Users can be a member of multiple organizations, and organizations can have a maximum of 3 users with the `owner` role. Users with the `owner` role can add and remove other organization owners over time, but organizations require at least one owner at all times.

> Organizations provide centralized governance through [access control](/hcp/docs/hcp/iam/access-management) and [enterprise constraints](/hcp/docs/hcp/admin/enterprise-constraints). Access control manages who can perform actions, while enterprise constraints restrict how HCP products can be used for cost control and compliance.

Reviewer: ```suggestion
Organizations provide centralized governance through [access control](/hcp/docs/hcp/iam/access-management) and [enterprise constraints](/hcp/docs/hcp/admin/enterprise-constraints). Access control manages who can perform actions, while enterprise constraints let organizations restrict HCP Product usage for cost control and compliance.
```

Slight rewording to use active voice
